components/slider/index.css

```diff
@@ -366,52 +388,55 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Slider-value {
-  margin-inline-start: var(--spectrum-slider-label-gap-x);
+  margin-inline-start: var(--mod-slider-label-margin-start, var(--spectrum-slider-label-margin-start));
 }
 
+/* Slider misses core-tokens for tick variant */
 .spectrum-Slider-ticks {
   display: flex;
   justify-content: space-between;
 
   z-index: 0;
 
-  margin: 0 var(--spectrum-slider-track-margin-offset);
-  margin-block-start: calc(
-    var(--spectrum-slider-tick-mark-height) +
-    1px
-  );
+  margin: 0 var(--mod-slider-track-margin-offset, var(--spectrum-slider-track-margin-offset));
+  margin-block-start: calc(var(--spectrum-slider-height-size) / 2 - var(--spectrum-slider-tick-mark-height) / 2);
+
+  /* missing core-token for handle fill-color */
+  &~.spectrum-Slider-handleContainer {
+    .spectrum-Slider-handle {
+      background: var(--mod-slider-ticks-handle-background-color, var(--spectrum-slider-ticks-handle-background-color));
+    }
+  }
 }
 
 .spectrum-Slider-tick {
   position: relative;
 
-  inline-size: var(--spectrum-slider-tick-mark-width);
+  inline-size: var(--mod-slider-tick-mark-width, var(--spectrum-slider-tick-mark-width));
```

Comment:
I think this can be the same token as the slider track thickness, like

```
.spectrum-Slider-tick {
  inline-size: var(--mod-slider-track-fill-thickness, var(--spectrum-slider-track-fill-thickness));
}
```

Reply:
I think we should keep this a separate token as we are adding the tick marks to the core-tokens package soon.